lua/astrocommunity/motion/harp-nvim/README.md

```diff
@@ -0,0 +1,15 @@
+# harp.nvim
+
+**Repositories:** 
+
+Harp: https://github.com/Axlefublr/harp (**Required to use the plugin**)
+
+Plugin: https://github.com/Thorinori/harp-nvim
+
+
+
+**Notes:**
+1. Plugin still in progress some (trying to not introduce any breaking changes to keybinds)
```

Comment:
This can be removed. It should be the readme of the plugin

Reply:
Removed. Figured some notes would make sense for a readme though since they are relevant to the plugin (unless you meant specifically that line)

Reply:
Well in general, if it's relevant to the plugin, it should be in the plugins readme. We do not want to maintain the burden of documenting the plugin or its behaviour, since it might change over time.

Reply:
Understandable. The main thing I really was aiming for there was the default binding related warnings, since that is more or less intended to be specific to the pack with the plugin not giving bindings by default on its own.